views/lib/sentry.hbs

```diff
@@ -0,0 +1,34 @@
+{{#ifneq @root.SENTRY_DSN false}}
+<script src="/vendor-optimized/@sentry/browser/build/bundle.min.js"></script>
+<script src="/vendor-optimized/@sentry/integrations/build/captureconsole.js"></script>
+<script>
```

Comment:
Maybe you can move all this code into a .js file and import the two sentry modules there and initialize it. Than we wouldn't need to copy the complete package with gulp and we could maybe load the script async. The way it is now, it blocks the rendering process and therefore increases the time to first paint.

Reply:
however there are variables to be replaced which will not work in plain js files :/ otherwise i really like that idea!

Reply:
Ahh haven't seen this.